x-pack/test_serverless/api_integration/test_suites/common/platform_security/authorization.ts

```diff
@@ -47,7 +47,8 @@ export default function ({ getService }: FtrProviderContext) {
       await supertestAdminWithApiKey.destroy();
     });
     describe('route access', () => {
-      describe('disabled', () => {
+      // skipped, see https://github.com/elastic/kibana/issues/194933
+      describe.skip('disabled', () => {
```

Comment:
@elastic/kibana-security This suite is failing because APIs are no longer disabled, I opened https://github.com/elastic/kibana/issues/194933 to track test migration from feature flag config to the regular one.

Reply:
Will be resolved by https://github.com/elastic/kibana/pull/195584